projects/packages/connection-ui/src/class-initial-state.php

```diff
@@ -45,7 +45,8 @@ private function can_use_connection_iframe() {
 		 * Filters whether the connection manager should use the iframe authorization
 		 * flow instead of the regular redirect-based flow.
 		 *
-		 * @since 8.3.0
+		 * @since 1.0.0
+		 * @since-jetpack 8.3.0
 		 *
```

Comment:
Since this filter is already [documented](https://github.com/Automattic/jetpack/blob/4ae507f6be6b83e7f96a5c7acb51c05b19609d79/projects/packages/connection/src/class-manager.php#L980) in the Connection package and the Connection UI package depends on the Connection package, should we consider removing the duplicate DocBlock in Connection UI and referring to the `connection/src/class-manager.php` file instead?

Reply:
It makes sense.

I'll only ping @sergeymitr here because I see that this method is not called anywhere. It's a private method and there's nothing in the class calling it. Should we remove it?

Reply:
Good catch, that's a leftover from when RNA Connection supported in-place flow, please remove it.
Also, please remove its copy from the Backup plugin: https://github.com/Automattic/jetpack/blob/update/connection-ui-annotations/projects/plugins/backup/src/php/class-initial-state.php#L39